addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -51,12 +55,31 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ChatterTopbar .o-active {
-    color: $o-brand-odoo;
-    background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
-    border-bottom-color: transparent;
-    border-top-color: transparent;
+.o_ChatterTopbar_actions.o-has-active-button {
+    border-bottom-color: $border-color;
+}
```

Comment:
If this work and has the same effect, it would be better
```suggestion
.o_ChatterTopbar_actions {
    border-color: $border-color;
}
```

Reply:
Don't know if already said it but, as seen hereunder, the border-bottom definition is more specific than border-color and implicitly border-bottom define the color as black, so no, it does not work.

![image](https://user-images.githubusercontent.com/48204996/82804241-3926c280-9e82-11ea-8122-046ebda8796b.png)

![image](https://user-images.githubusercontent.com/48204996/82804128-07adf700-9e82-11ea-911e-2507168a72a8.png)

Reply:
So you need the class specificity, but you may just use border-color and ONLY because he's after in the file (I think) so it override the border-bottom rule's color.
![image](https://user-images.githubusercontent.com/48204996/82804625-e1d52200-9e82-11ea-999b-5a174ce6135e.png)

Reply:
Ok I understand, so this issue is needed only because we split layout and style.

Reply:
Indeed